### PR TITLE
feat(migration): Chaging stuff after fork

### PR DESCRIPTION
**Pull request by carlosgit2016**
- Adjusted account
- Adjusted oidc role
- Adjusted workflows

### CODE REVIEW

File: .github/workflows/scaffold-cron.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ run-name: ${{ github.actor }}
 
 on:
   schedule:
-    - cron: "30 9 * * 1,5"   # Run on Weekdays at 9:30AM UTC
+    - cron: "00 01 * * 1,5"   # Run on Weekdays at 1AM UTC
```
Comment (benhur-araujo) [inline .github/workflows/scaffold-cron.yaml]: - cron: "00 01 * * 1,2,3,4,5"  # Run on Weekdays at 1AM UTC

File: .github/workflows/vanish.yml [hunk 2]
```diff
@@ -2,11 +2,9 @@ name: vanish
 run-name: "Destroying in case we forgot"
 on:
   schedule:
-    - cron: "0 13 * * 1,5"  # Every 1PM UTC Mon-Fri
-    - cron: "0 23 * * SAT,SUN"    # Every SAT-SUN at 11PM UTC
+    - cron: "0 05 * * 1,5"  # Every 5AM UTC Mon-Fri
```
Comment (benhur-araujo) [inline .github/workflows/vanish.yml]: - cron: "0 05 * * 1,2,3,4,5"  # Every 5AM UTC Mon-Fri

File: README.md
Comment (benhur-araujo) [inline README.md:1]: NICE!

File: utils/root_get_temporary_token.sh
Comment (benhur-araujo) [inline utils/root_get_temporary_token.sh:1]: Nice! What would be a use case for this in our project?

**Review by benhur-araujo — APPROVED** [review]

File: utils/root_get_temporary_token.sh
Comment (carlosgit2016) [reply utils/root_get_temporary_token.sh:1]: Initially I though in using it to plan/apply initial terraform. So it is not necessary to create a user and credentials to it.

But then I configured Identity Center that is a SSO and I was able to generate temporary credentials through it. So it is not necessary to have a user with long lived Tokens to scaffold the initial infrastructure

File: .github/workflows/scaffold-cron.yaml (hunk 1 above)
Comment (carlosgit2016) [reply .github/workflows/scaffold-cron.yaml]: nice catch, I think this is the same thing, but short `00 01 * * 1-5`

**Pull request merged**